portfolio/src/main/webapp/script.js

```diff
@@ -368,6 +370,26 @@ async function loadComments() {
   return await request.json();
 }
 
+/**
+ * Sends a request to delete all comments, and 
+ * re-renders the comments list.
+ */
+async function deleteComments() {
+  await requestCommentDeletion()
+    .then(loadComments)
```

Comment:
curious: why do we need to call `loadComments` after we delete them?

Reply:
It's part of the walk-through guide. It says "*Note: it might look a little strange to make a GET request to `/data` when we know we will just be deleting all the comments, but it's usually a good idea to let the server be the single source of truth. This prevents our web page from getting out of sync with the datastore as we add more complex behavior.*"

Reply:
Cool, thanks for clarifying